Dockerfile.erb

```diff
@@ -26,7 +26,7 @@ RUN set -ex \
 	&& gpg --verify bitcoin.asc \
 <% end -%>
 	&& tar -xzvf $BITCOIN_DIST -C /usr/local --strip-components=1 --exclude=*-qt \
-	&& rm bitcoin*
+	&& rm -f bitcoin* BUcash*
```

Comment:
I don't like that we need to change this for every existing dockerfile. Does the new bitcoin cash version untar to this directory, or is the tar file named this? It's probably cleaner if we just start specifying an exact name for the tar file we download.